src/types/ant.ts

```diff
@@ -55,8 +55,13 @@ export const AntKeywordsSchema = z.array(z.string()); // TODO: add specific limi
 export const AntRecordSchema = z.object({
   transactionId: ArweaveTxIdSchema.describe('The Target ID of the undername'),
   ttlSeconds: z.number(),
+  priority: z.number().optional(),
 });
 export type AoANTRecord = z.infer<typeof AntRecordSchema>;
+export type ANTRecords = Record<string, AoANTRecord>;
+export type SortedANTRecord = AoANTRecord & { index: number };
+export type SortedANTRecords = Record<string, SortedANTRecord>;
```

Comment:
Nit: It's nice when the key type is a type alias that describes the domain of the key string.